mount_config.go

```diff
@@ -142,6 +142,8 @@ type MountConfig struct {
 	// For expert use only! May invalidate other guarantees made in the
 	// documentation for this package.
 	Options map[string]string
+
+	SubType string
```

Comment:
Please name this Subtype.

Reply:
What do you mean about 'name this Subytpe'?